Exercise_4/css/style.css

```diff
@@ -0,0 +1,204 @@
+    border-bottom: 0.1rem solid lightgrey;
+    border-top: 0.2rem solid lightgrey;
+    display: flex;
+    height: 3rem;
+    left: 0;
+    margin: 2rem 0;
+    padding: 1rem 0;
+    position: fixed;
+    right: 0;
+    top: 0;
+}
+
+.header__title {
+    font-size: 3rem;
+    margin: 0 auto;
+    padding-right: 10vw;;
+}
+
+/*HEADER SIDE-MENU*/
+.side-menu {
+    display: none;
+}
+
+/*SELECT NAVIGATION*/
+.select-navigation {
+    display: none;
+}
+
+@media only screen and (max-width: 880px) {
+    /*HEADER SIDE-MENU*/
+    .side-menu {
+        cursor: pointer;
+        display: flex;
+        font-size: 2rem;
+        margin: 0 2rem;
+        width: 10vw;
+    }
+
+    .side-menu .side-menu__content {
+        display: none;
+        padding: 1rem;
+    }
+
+    .side-menu:focus .side-menu__content {
+        background-color: white;
+        box-shadow: 1rem 2rem 3rem darkgrey;
+        display: flex;
+        flex-direction: column;
+        height: 100vh;
+        left: 0;
+        min-width: 20rem;
+        overflow-y: auto;
+        position: fixed;
+        top: 0;
+        width: 30vw;
+        z-index: 1;
+    }
+
+    .side-menu__content .search-bar {
+        background-color: white;
+        box-sizing: border-box;
+        font-family: FontAwesome;
+        font-size: 2rem;
+        margin: 0.2rem 0.2rem 0 1rem;
+        padding: 0.2rem;
+        text-align: end;
+    }
+
+    .side-menu__content .list {
+        list-style-type: none;
+        padding: 0;
+    }
+
+    .side-menu__content .list-item {
+        border-bottom: 0.1rem solid lightgrey;
+        font-size: 1.5rem;
+        margin: 0 1rem;
+        padding: 1rem;
+    }
+
+    .side-menu__content .list .list-title {
+        font-size: 2rem;
+        font-weight: bold;
+        margin-left: 1rem;
+    }
+
+    .side-menu__content .list:last-of-type .list-item {
+        border-bottom: none;
+        color: grey;
+        padding: 1.2rem 0 0 0;
+    }
+
+    .side-menu__content .footer {
+        align-items: center;
+        display: flex;
+        justify-content: center;
+        margin-top: 1rem;
+    }
+
+    .side-menu__content .footer i {
+        font-size: 2rem;
+    }
+
+    /*SELECT NAVIGATION*/
+    .select-navigation {
+        display: flex;
+        justify-content: center;
+        margin: 8rem auto 0 auto;
+        font-size: 2rem;
+        width: 80vw;
+    }
+
+    .select-navigation__item {
+        font-size: 1.5rem;
+        padding: 0.3rem 0;
+    }
+}
+
+/*MENU*/
+.menu-registration {
+    display: none;
+}
+
+.menu {
+    display: none;
+}
+
+/*NAVIGATION*/
+.navigation {
+    display: none;
+}
+
+@media only screen and (min-width: 880px) {
+    /*MENU*/
+    .menu {
+        display: flex;
+        margin: 0 5rem;
+        width: 30vw;
+    }
+
+    .menu__item {
+        color: black;
+        font-size: 1.2rem;
+        font-weight: bold;
+        padding: 0 1rem;
+        text-decoration: none;
+    }
+
+    .menu-registration {
```

Comment:
There is an issue with "Sign In" on some screen sizes. Please check the picture: https://snag.gy/tnA5pW.jpg

To fix it you simply can use `&nbsp;` instead of a space or try to find another solution with word wrapping or smth.